sql/mysqld.cc

```diff
@@ -4640,36 +4640,6 @@ a file name for --log-bin-index option", opt_binlog_index_name);
     mysql_mutex_unlock(log_lock);
   }
 
-<<<<<<< HEAD
-#ifdef HAVE_REPLICATION
-  if (opt_bin_log && expire_logs_days)
-  {
-    time_t purge_time= server_start_time - expire_logs_days*24*60*60;
-    if (purge_time >= 0)
-      mysql_bin_log.purge_logs_before_date(purge_time, true);
-  }
-  if (opt_bin_log && max_binlog_files)
```

Comment:
Not sure if this is Ok to move these two conditions from here...

Reply:
This was tricky. Please, mind that mysql moved the block these two are included in.
<https://github.com/mysql/mysql-server/commit/37b047220a907c2a6d7235ddf2b7a6be916cc82e>